packages/xpetra/src/MultiVector/Xpetra_EpetraIntMultiVector.cpp

```diff
@@ -127,8 +133,8 @@ template class EpetraIntMultiVectorT<long long, Kokkos::Compat::KokkosOpenMPWrap
 template Epetra_IntMultiVector & toEpetra<long long,Kokkos::Compat::KokkosOpenMPWrapperNode >(MultiVector<int, int, long long, Kokkos::Compat::KokkosOpenMPWrapperNode> &);
 template const Epetra_IntMultiVector & toEpetra<long long, Kokkos::Compat::KokkosOpenMPWrapperNode >(const MultiVector<int, int, long long, Kokkos::Compat::KokkosOpenMPWrapperNode> &);
 #endif
-#ifdef HAVE_TPETRA_INST_CUDA
-typedef Kokkos::Compat::KokkosCudaWrapperNode default_node_type;
+#ifdef HAVE_TPETRA_INST_HIP
```

Comment:
Why was the HAVE_TPETRA_INST_CUDA clause removed?

Reply:
Also fixed